docs/src/main/sphinx/security/overview.rst

```diff
@@ -20,31 +20,66 @@ can be enabled for different parts of the Trino architecture:
 Suggested configuration workflow
 --------------------------------
 
-To configure security for a new Trino cluster, follow this best practice
-order of steps. Do not skip or combine steps.
+To configure security for a new Trino cluster, follow this best practice order
+of steps. Each step is followed by an additional verification step. Do not skip
+or combine steps.
 
-#. **Enable** :doc:`TLS/HTTPS </security/tls>`
+1. **Enable** :doc:`TLS/HTTPS </security/tls>`
 
    * Work with your security team.
    * Use a :ref:`load balancer or proxy <https-load-balancer>` to terminate
      HTTPS, if possible.
    * Use a globally trusted TLS certificate.
 
-#. **Configure** a :doc:`a shared secret </security/internal-communication>`
+To verify this step, connect to the :doc:`Web UI </admin/web-interface>` and
+send a query with the Trino :doc:`CLI </client/cli>` using a URL that begins
+with ``HTTPS``. If successfully enabled, you will not be able to send a query or
+use the Web UI with a URL that begins with ``HTTP``.
```

Comment:
http will still work unless the steps talk about disabling that .. which could be a separate step .. read up in linked doc